tools/core_packages.txt

```diff
@@ -76,7 +76,6 @@ perl_text_unidecode
 perl_unicode_eastasianwidth
 perl_xml_parser
 pkgconfig
-python27
```

Comment:
Python 2 is not going away any time soon.  Too many apps still depend on it with no upgrade path to Python 3.

Reply:
@satmandu: How can we restore python27 and python3 back to core?

Reply:
See https://github.com/skycocker/chromebrew/pull/5243